metadata.json

```diff
@@ -97,6 +97,9 @@
       ]
     },
     {
+      "operatingsystem": "Archlinux"
```

Comment:
It seems to, yes. I ran a full spec (no ONLY_OS exported) before starting on the tests and got a bunch of failures.